scripts/import_mobile_keys

```diff
@@ -28,10 +26,14 @@ function cp_from_keepass {
   cp "$path" "$2"
 }
 
-check_dir
+
+COZY_DIRECTORY=${COZY_DIRECTORY:-""}
+PASSWORD_STORE_PATH=${PASSWORD_STORE_PATH:-"$HOME/.password-store/${COZY_DIRECTORY}"}
+BANKS_DIR=${PASSWORD_STORE_PATH}cozy-banks
```

Comment:
```suggestion
BANKS_DIR_IN_PASSWORD_STORE=${PASSWORD_STORE_PATH}cozy-banks
```
?

At first I thought the cozy-banks' code directory should be contained within password store path

Reply:
💯 Fixed